public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -169,6 +170,10 @@ export function useAllAlertmanagerAbilities(): Abilities<AlertmanagerAction> {
       ctx.hasPermission(notificationsPermissions.provisioning.read) ||
         ctx.hasPermission(notificationsPermissions.provisioning.readSecrets),
     ],
+    [AlertmanagerAction.CanReadSecrets]: [
+      hasConfigurationAPI,
```

Comment:
I think this action should be only supported by Grafana Alertmanager, yet I'm not sure if we should have this action as a separate one because it's meaningless from the UI perspective

Reply:
true